### ISSUE TITLE
HTTP-TS: Support mp3 audio codec.

### ISSUE THREAD

**Issue by winlinvip** [xqq/mpegts.js#83]
If audio is mp3 codec, play is ok but no audio. Replay steps is bellow.

1. Start SRS `v4.0.269+` server:

```bash
./objs/srs -c conf/mp3.ts.conf
```

2. Publish RTMP stream with mp3 audio:

```bash
ffmpeg -stream_loop -1  -re -i doc/source.flv -vcodec copy -acodec libmp3lame \
    -f flv rtmp://localhost/live/livestream
```

3. Play HTTP-TS stream:

* URL is `http://localhost:8080/live/livestream.ts`
* VLC and ffplay is OK.
* The [srs-player](http://localhost:8080/players/srs_player.html?stream=livestream.ts) fail, which use mpegts.js

PS: If use HLS with mp3 audio, hls.js is ok to play.

**Comment by xqq** [comment]
Implemented in #86 by @monyone

**Comment by winlinvip** [comment]
It works! Thanks a lot 👍 

<img width="720" alt="image" src="https://user-images.githubusercontent.com/2777660/210169916-9dfe9258-ac75-4de6-8aad-35ba73161dda.png">
